### PR TITLE
Add i18n 

### LINKED ISSUE #20: Add support for multiple languages
When I started the `BTB-manager-telegram` project my goal was to give easy access to autmated trading to people who are not familiar with terminals and such.  
I had chats with many of this project's users and many of them don't seem to be native english speakers.  
I believe adding mulitple language support will be another step into the right direction and will provide even easier access to automated trading technology to people from anywhere in the world.

### PR DESCRIPTION
**Pull request by Lisiadito**
Fixes #20 

I have a german translation in progress as well but I wanted to get this reviewed before putting to much effort in the german translation.

### CODE REVIEW

**Comment by lorcalhost** [comment]
Hi @Lisiadito , I see that you chose to include a file `./config/lang.cfg` for language selection.  
Changing values inside this file would interfere with the update-process (the update button runs `git pull`).  
I think a better solution should be found for language selection:
- CLI flags
- Manually created file / created on first startup
- Langauge selection through bot menus
- Other ways (?)

Other than that thanks for the PR, I'll check it out more throughly when I have time

**Comment by Lisiadito** [comment]
Ah I didn't think about that. Makes sense. Yeah will think about it. Do you have a preference? 
Tomorrow I will start using that branch and test if I actually missed something 😁

**Comment by lorcalhost** [comment]
No preference, each option I listed has its pros and cons.  
Feel free to suggest alternatives.

**Comment by Lisiadito** [comment]
@lorcalhost I still find some mistakes I made and I'll fix them the next days. I let you know when the branch is ready for a review. And I can squash all the bugfix commits when I'm done.

**Comment by Lisiadito** [comment]
@lorcalhost Ok I fixed everything I've found. I use this branch now with my bot so I'll see if anything more shows up but I would say it's ready for review.

**Comment by Lisiadito** [comment]
Obviously I still have to decide for a solution on how to set the language.

**Comment by lorcalhost** [comment]
Thank you for your hard work!

File: i18n/en.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+sure_delete: "Are you sure you want to delete the database file and clear the logs?"
+stop_bot_before_edit_config: '⚠ Please stop Binance Trade Bot before editing user configuration file\.'
+config_file_is: "Current configuration file is:"
+reply_config: '_*Please reply with a message containing the updated configuration*_\.'
+stop_to_stop: 'Write /stop to stop editing and exit without changes\.'
+config_file_error: '❌ Unable to find user configuration file at `%{path}`\.'
+stop_bot_before_edit_coin_list: '⚠ Please stop Binance Trade Bot before editing the coin list\.'
+coin_list_is: "Current coin list is:"
+reply_coin_list: '_*Please reply with a message containing the updated coin list*_\.'
+coin_list_error: '❌ Unable to find user configuration file at `%{path}`\.'
```
Comment (shhhmel) [inline i18n/en.yml:57]: `user configuration file` -> `coin list`

File: i18n/en.yml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+coin_edit_error: '❌ Unable to edit coin list file\.'
+exited_no_change: '👌 Exited without changes\.'
+coin_list_not_modified: 'Your `supported_coin_list` file was *not* modified\.'
+edited_user_config: "✔ Successfully edited user configuration file to:"
+user_config_error: '❌ Unable to edit user configuration file\.'
+user_config_not_modified: 'Your `user.cfg` file was *not* modified\.'
+deleted_db: '✔ Successfully deleted database file\.'
+delete_db_error: '❌ Unable to delete database file\.'
+clear_log_error: '❌ Unable to clear log file\.'
+exit_no_change: '👌 Exited without changes\.'
```
Comment (shhhmel) [inline i18n/en.yml:136]: Duplicate of line 128

File: i18n/en.yml (hunk 1 above)
Comment (Lisiadito) [reply i18n/en.yml:57]: fixed

File: i18n/en.yml (hunk 2 above)
Comment (Lisiadito) [reply i18n/en.yml:136]: fixed

**Comment by Lisiadito** [comment]
@lorcalhost soooo finally managed to fix the way the language is selected. Decided for the command line flag. Sorry for the crazy delay I was super busy.

@shhhmel thanks to you for reviewing and finding some issues. You can merge this branch now again into yours. You need to update some stuff:
Readme, options for the commandline flag help text and I removed one and added one keyword to the en.yml file.
But you will see.

**Comment by lorcalhost** [comment]
@Lisiadito thank you for working on this even after all this time.
I'll check out the update

**Comment by Lisiadito** [comment]
Oh I just saw that I missed the next_coin funtion. I came in with the merge of the main branch. I add i18n to that and push it in a few minutes

**Comment by Lisiadito** [comment]
I fixed the order of the next_coin results. was just one line and it was bothering me a lot.

**Comment by titulebolide** [comment]
@Lisiadito is this PR ready to be merged according to you? If so I will start to check it :)

**Comment by titulebolide** [comment]
@Lisiadito up :) i would like to have your advice on it before merging

File: btb_manager_telegram/buttons.py [hunk 3]
```diff
@@ -336,12 +350,13 @@ def next_coin():
                 query = cur.fetchall()
 
                 m_list = []
+                query = sorted(query, key=itemgetter(3), reverse=True)
```
[the remark this answers is not in the document]
Comment (Lisiadito) [reply btb_manager_telegram/buttons.py:353]: Hey sorry I didn't get a notification that you commented. I don't think there was a specific reason for using that way other than that I'm not fluent in python. I can change it and test if it behaves the same.

[the remark this answers is not in the document]
Comment (titulebolide) [reply btb_manager_telegram/buttons.py:353]: Hey thanks for your reply @Lisiadito :)
Using lambda function in this case makes things clearer to me, as itemgetter is not common to be seen in python
Moreover it removes an aditionnal import

[the remark this answers is not in the document]
Comment (Lisiadito) [reply btb_manager_telegram/buttons.py]: Changed it.

**Comment by titulebolide** [comment]
@Lisiadito did you had the change tested in any case?

**Comment by titulebolide** [comment]
Great work. Thanks for your contribution @Lisiadito !

**Comment by titulebolide** [comment]
@all-contributors please add @Lisiadito for code, idea, translation, doc

**Comment by titulebolide** [comment]
@all-contributors please add @Lisiadito for code, idea, translation, doc

**Pull request merged**